package-scripts/datadog-agent/postinst

```diff
@@ -3,6 +3,8 @@
 CONFIG_DIR=/etc/dd-agent
 LOG_DIR=/var/log/datadog
 RUN_DIR=/opt/datadog-agent/run
+JMX_STATUS_PYTHON=/tmp/jmx_status_python.yaml
+JMX_STATUS=/tmp/jmx_status.yaml
```

Comment:
I'm not sure it would work, because this script is run as `root`, and the agent as `dd-agent`, so in this specific case `tempfile.gettempdir()` won't work. And even if we use `sudo` or `su`, some env variables won't be there...
I'm not sure it's worth spending too much time on it, it's really a small fraction of users. Maybe just warn support about it ?